pkg/dispatchcli/cmd/create.go

```diff
@@ -158,6 +171,7 @@ func NewCmdCreate(out io.Writer, errOut io.Writer) *cobra.Command {
 				"BaseImage": CallCreateBaseImage,
 				"Function":  CallCreateFunction,
 				"Secret":    CallCreateSecret,
+				"Policy":    CallCreatePolicy,
```

Comment:
While you are at it can you make this `utils.PolicyKind` and similar change to other keys?

Reply:
fixed, constants replaced with utils.*Kind. Include create.go and delete.go